plugins/pulp_rpm/plugins/migrations/0023_fix_translated_fields_type.py

```diff
@@ -0,0 +1,36 @@
+"""
+This migration modifies translated_name and translated_description.
+
+When translated_name or translated_description are set to empty string, This migration sets them
+to {}. This is required since empty string will not validate with the Mongoengine definition.
+"""
+from pulp.server.db import connection
+
+
+def fix_translated_fields_string_to_dict(collection):
+    """
+    Change translated_name and translated_description fields to {} if they are "".
+
+    :param collection: The collection to have its translated fields fixed up
+    :type  collection: pymongo.Collection
+
+    :rtype: None
+    """
+    collection.update({"translated_name": ""}, {'$set': {'translated_name': {}}})
```

Comment:
I think both of these calls to ``update()`` need the argument ``multi=True``

https://api.mongodb.org/python/2.8/api/pymongo/collection.html#pymongo.collection.Collection.update